bin/create-scripts/block.js

```diff
@@ -0,0 +1,59 @@
+/** @format */
```

Comment:
Let's add a hashbang (`#!/usr/bin/env node`) and make this file executable (so we can call the script without having to prepend `node`).

Reply:
🤷‍♂️ 

I guess I don't care. I tested that `node` still works when invoked directly (even though the hash-bang isn't valid JS)

my preference is not to hard-encode system-specific directives in sourcecode but it's not a strong preference. for this PR let's leave it out. you can add it in another PR afterwards.